lib/Bindings/Python/circt/design_entry.py

```diff
@@ -83,3 +83,11 @@ def body_build(mod):
           body_builder=body_build)
 
   return __Module
+
+
+def connect(destination, source):
+  builder = destination.builder
+  index = destination.index
+  builder.operation.operands[index] = source.value
```

Comment:
`source` should probably accept `Values`, `Operations` (with one result), and `BuilderValues`.